Data/ByteString.hs

```diff
@@ -40,8 +40,9 @@
 
 module Data.ByteString (
 
-        -- * The @ByteString@ type
+        -- * The strict @ByteString@ type its associated type alias
```

Comment:
I think a reader might wonder, what "strict" means in this context. Do we have an answer for this?

In the case of the `StrictByteString` name, I mean "strict" as opposed to "lazy".

Also, there seems to be an "and" missing before "its associated …". (Similarly in `D.B.Lazy`)

Reply:
> Also, there seems to be an "and" missing before "its associated …". (Similarly in D.B.Lazy)

oof, this isn't good. Thanks.

> I think a reader might wonder, what "strict" means in this context. Do we have an answer for this?

I have a hard time conceiving that the reader does not know what it strictness in the context of lazy functional programming, and should this happen, it ought to be fixed upstream, in pedagogical material. 

Now we could also be open to feedback from our readers and if it seems that people are excessively confused (compared to what is acceptable), then we would change the wording.

Reply:
If you reasonably think this will be a problem, I can phrase it as such:

> The strict (or non-lazy) @ByteString@ type and its associated type alias

Reply:
> > I think a reader might wonder, what "strict" means in this context. Do we have an answer for this?
> 
> I have a hard time conceiving that the reader does not know what it strictness in the context of lazy functional programming, and should this happen, it ought to be fixed upstream, in pedagogical material.

Yeah, good point. I guess the usual definition is that WHNF for such a type implies NF.